docs/csharp/language-reference/operators/patterns.md

```diff
@@ -96,19 +96,19 @@ Beginning with C# 9.0, you use the `not`, `and`, and `or` pattern combinators to
 
 - *Negation* `not` pattern that matches an expression when the negated pattern doesn't match the expression. For example, you can negate the [constant](#constant-pattern) `null` pattern to check if an expression is non-null, as the following code shows:
 
-  :::code language="csharp" source="snippets/patterns/PatternCombinators.cs" id="NotPattern":::
+  :::code language="csharp" source="snippets/patterns/LogicalPatterns.cs" id="NotPattern":::
 
 - *Conjunctive* `and` pattern that matches an expression when both patterns match the expression. For example, you can combine [relational patterns](#relational-patterns) to check if a value is in a certain range, as the following code shows:
 
-  :::code language="csharp" source="snippets/patterns/PatternCombinators.cs" id="AndPattern":::
+  :::code language="csharp" source="snippets/patterns/LogicalPatterns.cs" id="AndPattern":::
```

Comment:
This snippet isn't rendering in the review site. Is the snippet ID incorrect?

Reply:
Thank you @BillWagner 
The snippet has not been closed. I've committed the fix. Does it render now?